platform/authorized_api/users/index.json

```diff
@@ -147,8 +158,18 @@
                   "$ref": "#/definitions/User_Password"
                 },
                 "email": {
-                  "type": "string",
-                  "format": "email"
+                  "$ref": "#/definitions/User_Email"
+                },
+                "role": {
+                  "allOf": [
+                    {
+                      "$ref": "#/definitions/User_Role"
```

Comment:
FYI: Names of definitions are used in documentation:
![screenshot from 2018-11-19 16-35-36](https://user-images.githubusercontent.com/4488774/48717261-550c3100-ec19-11e8-8e5e-64e15a69b0fd.png)
It may be good idea to rethink them.

Reply:
thanks, we will talk about that